dvclive/metrics.py

```diff
@@ -61,23 +61,53 @@ def _cleanup(self):
             os.remove(self.html_path)
 
     @staticmethod
-    def from_env():
+    def from_env(existing_logger: "MetricLogger" = None):
```

Comment:
@pared I'm not sure what the changes to `from_env` are intended to do. `from_env` is never being called with the new arg `existing_logger`. Could these changes be from previous iterations and are now outdated?

Reply:
Yep, wrongly conducted rebase, sorry for that